src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -0,0 +1,56 @@
+package se.kth.sda.skeleton.comments;
+
+import javax.persistence.*;
+import java.util.Date;
+
+/**
+ * This class is responsible for shaping comment as entity.
+ */
+@Entity
+@Table(name="comments")
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(columnDefinition = "TEXT")
+    private String body;
+
+    @Column(columnDefinition = "TEXT")
+    private Date date;
+
+    // One post can have multiple entities
+    @ManyToOne
+    Post post;
```

Comment:
should we make the post private ? then it can only be changed in the public setter named setPost(args)